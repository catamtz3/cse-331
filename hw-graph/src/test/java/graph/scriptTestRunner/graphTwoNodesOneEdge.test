# Create a graph
CreateGraph twoNodes

#Add two nodes to the graph
AddNode twoNodes a
AddNode twoNodes b

#Add one edge between nodes a and b
AddEdge twoNodes ab a b

# Get the nodes of the graph
ListNodes twoNodes