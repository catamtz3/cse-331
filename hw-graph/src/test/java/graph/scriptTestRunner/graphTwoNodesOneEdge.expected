# Create a graph
Created graph twoNodes

#Add two nodes to the graph
added node a to twoNodes
added node b to twoNodes

#Add one edge between nodes a and b
added edge ab from a to b in twoNodes

# Get the nodes of the graph
twoNodes contains: a[ab], b